ci/ci

```diff
@@ -508,7 +508,8 @@ def build_compose_file(nodes):
     for n in nodes.split(';'):
         count = count + 1
         node_key, node_val = n.split('=')
-        if node_val not in MACROS['SUPPORTED_PLATFORMS'].split(','):
+        if node_val not in MACROS['SUPPORTED_PLATFORMS'].split(',') \
+                and ''.join(sys.argv).find(node_val) != -1:
```

Comment:
Did you mean '' without a space, or ' ' with a space?

Reply:
@bhroam it doesn't matter actually, we just need to see that word (which is value of `node_val`) is in sys.argv or not.